Files/UserControls/StatusBarControl.xaml

```diff
@@ -234,23 +234,32 @@
             </Button>
 
             <Grid x:Name="StatusCenterGrid" Visibility="{x:Bind ShowStatusCenter, Mode=OneWay}">
-                <muxc:AnimatedVisualPlayer
-                    x:Name="StatusCenterPulseVisualPlayer"
-                    AutoPlay="False"
-                    Visibility="Collapsed">
-                    <lottie:LottieVisualSource x:Name="StatusCenterPulseVisualSource" UriSource="ms-appx:///Assets/AnimatedVisuals/LottiePulseStatusCenter.json" />
-                </muxc:AnimatedVisualPlayer>
                 <Button
                     x:Name="StatusCenter"
                     x:Uid="StatusCenter"
                     Width="36"
                     Height="32"
+                    Padding="0"
                     AutomationProperties.Name="Status center"
                     Background="Transparent"
                     Style="{StaticResource ToolBarButtonStyle}"
                     ToolTipService.ToolTip="Ongoing Tasks"
                     Visibility="Visible">
-                    <FontIcon FontSize="16" Glyph="&#xF16A;" />
+                    <Button.Content>
+                        <Grid>
+                            <FontIcon
+                                x:Name="StatusCenterFontIcon"
+                                FontSize="16"
+                                Glyph="&#xF16A;"
+                                x:Load="{x:Bind AnyOperationsOngoing, Converter={StaticResource BoolNegationConverter}, Mode=OneWay}" />
+
+                            <muxc:ProgressRing
+                                Width="22"
+                                Height="22"
+                                IsActive="True"
+                                Visibility="{x:Bind AnyOperationsOngoing, Mode=OneWay}" />
```

Comment:
`AnyOperationsOngoing` is a bool, not a `Visibility`. Please use a converter or remove AnyOperationsOngoing from this. Or use x:Load